modules/nf-core/samtools/coverage/meta.yml

```diff
@@ -0,0 +1,45 @@
+name: "samtools_coverage"
+description: produces a histogram or table of coverage per chromosome
+keywords:
+  - coverage
+  - samtools
```

Comment:
Can you change the tool name to something else here? (doesn't really matter what, as long as it belong to this tool :))